src/libraries.js

```diff
@@ -158,9 +158,84 @@ export const BANNED_LIBRARIES = [
   'angularjs.1.5.4.angular.min.js',
   'angularjs.1.5.5.angular.js',
   'angularjs.1.5.5.angular.min.js',
+  'angularjs.1.5.6.angular.js',
+  'angularjs.1.5.6.angular.min.js',
 ];
 
 export const UNADVISED_LIBRARIES = [
-  'angularjs.1.5.6.angular.js',
```

Comment:
Why do these get labelled as unadvised as soon as the new ones come out? That seems quite aggressive for a bugfix release.